### ISSUE TITLE
DataFrames required but not installed when running serve for first time

### ISSUE THREAD

**Issue by KristofferC** [tlienart/PkgPage.jl#33]
After doing `serve()` I got

```
┌ Franklin Warning: in <index.md>
│ There was an error of type 'ArgumentError' when running a code block.
│ Checking the output files '/Users/kristoffercarlsson/Documents/JuliaPkgs/LazilyInitializedFields/page/__site/assets/index/code/output/ex.(out|res)'
│ might be helpful to understand and solve the issue.
```

and looking in the file `ex.out` I see

```
ArgumentError: Package DataFrames [a93c6f00-e57d-5684-b7b6-d8193f3e46c0] is required but does not seem to be installed:
 - Run `Pkg.instantiate()` to install all recorded dependencies.
```

**Comment by KristofferC** [comment]
I think maybe a `Pkg.instatiate` is needed after activating the new project in `page`.

**Comment by tlienart** [comment]
I agree there should be some automation here (there is some but I need to revisit this). The docs does kind of say you're supposed to do it yourself to ensure things work fine but I agree it's not ideal; thanks for the heads up

![Screenshot 2020-10-05 at 11 24 20](https://user-images.githubusercontent.com/10897531/95062545-6cfdd880-06fd-11eb-876d-783f8ba56929.png)

(PS: this is tied to Franklin so I need to figure out how to fix it there in a more general way)

**Comment by KristofferC** [comment]
Oh, I guess I missed it in the docs because I got the error when I executed `serve` which is shown before the section about showing code.

**Comment by tlienart** [comment]
Ah dayum it _activates_ it but it does not _instantiate_ it, that's really dumb...
